docs/non-normative/groups/system/use-cases.md

```diff
@@ -0,0 +1,77 @@
+# **System Semantic Conventions: General Use Cases**
+
+This document is a collection of the use cases that we want to cover with the System Semantic Conventions. The use cases outlined here inform the working group’s decisions around what instrumentation is considered **required**.  
+Use cases in this document will be stated in a generic way that does not refer to any potentially existing instrumentation in semconv as of writing, such that when we do dig into specific instrumentation, we understand their importance based on our holistic view of expected use cases.
+
+## *Legend*
+
+`General Information` \= The information that should be discoverable either through the entity, metrics, or metric attributes.
+
+`Dashboard` \= The information that should be attainable through metrics to create a comprehensive dashboard.
+
+`Alerts` \= Some examples of common alerts that should be creatable with the available information.
+
+## **Host**
+
+A user should be able to monitor the health of a host, including monitoring resource consumption, unexpected errors due to resource exhaustion or malfunction of core components of a host or fleet of hosts (network stack, memory, CPU…).
+
+### General Information
+
+* Machine name  
+* ID (relevant to its context, could be a cloud provider ID or just base machine ID)  
+* OS information (platform, version, architecture, etc)  
+* Number of CPU cores  
```

Comment:
Maybe this can be "CPU information" instead? We have a bunch of those [here](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/host.md)